src/coord/src/error.rs

```diff
@@ -124,6 +126,9 @@ impl CoordError {
             }
             CoordError::Catalog(c) => c.detail(),
             CoordError::Eval(e) => e.detail(),
+            CoordError::InvalidTableMutationSelection => Some(
+                "mutation operations on tables may only refer to other user-created tables".into(),
```

Comment:
Thoughts about making this the error text instead? It's not too long and is more actionable than the current text. Some drivers might not surface the detail text and users could remain confused with just the below text.